src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -2894,18 +2894,10 @@ public Place createDeletePlace() {
 
   @Override
   public void onShare() {
-    if (
-      featureFlagConfig.isFeatureEnabled(FeatureFlagKey.REACT_ENTITY_ACL_EDITOR)
-    ) {
-      getEntityAccessControlListModalWidget()
-        .configure(entity.getId(), this::fireEntityUpdatedEvent);
-      getEntityAccessControlListModalWidget().setOpen(true);
-    } else {
-      getAccessControlListModalWidget()
-        .configure(entity, permissions.getCanChangePermissions());
-      this.getAccessControlListModalWidget()
-        .showSharing(this::fireEntityUpdatedEvent);
-    }
```

Comment:
The feature flag is currently `true` (we want to use the feature). Shouldn't we preserve the `if` block, not `else`?